keel-core/src/main/kotlin/com/netflix/spinnaker/keel/veto/Veto.kt

```diff
@@ -28,32 +28,32 @@ import com.netflix.spinnaker.keel.api.Resource
 interface Veto {
 
   /**
-   * The name of the veto
+   * The name of the veto.
    */
   fun name(): String = javaClass.simpleName
 
   /**
-   * Check whether the resource can be checked according to this veto
+   * Check whether the resource can be checked according to this veto.
    */
-  fun check(resource: Resource<*>): VetoResponse
+  suspend fun check(resource: Resource<*>): VetoResponse
```

Comment:
This method is always called from a suspending context, so we might as well allow implementations to freely use coroutines.